groupcache.go

```diff
@@ -188,15 +195,16 @@ type flightGroup interface {
 
 // Stats are per-group statistics.
 type Stats struct {
-	Gets           AtomicInt // any Get request, including from peers
-	CacheHits      AtomicInt // either cache was good
-	PeerLoads      AtomicInt // either remote load or remote cache hit (not an error)
-	PeerErrors     AtomicInt
-	Loads          AtomicInt // (gets - cacheHits)
-	LoadsDeduped   AtomicInt // after singleflight
-	LocalLoads     AtomicInt // total good local loads
-	LocalLoadErrs  AtomicInt // total bad local loads
-	ServerRequests AtomicInt // gets that came over the network from peers
+	Gets                        AtomicInt // any Get request, including from peers
+	CacheHits                   AtomicInt // either cache was good
+	GetFromPeersSlowestDuration AtomicInt // slowest duration to request value from peers
```

Comment:
I would call it `GetFromPeersLatencyLower` to be similar to how Graphana reports things.

Reply:
done :)